Notes/2018-11-15.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>November 15, 2018}</title>
```

Comment:
Looks like you have a typo here
```suggestion
    <title>November 15, 2018</title>
```

Reply:
Ok I fixed it.  Thank you for catching that.